### PR TITLE
feature/add status for tickets #388

### PR DESCRIPTION
**Pull request by nmate980829**

### CODE REVIEW

File: migrations/20200828113656_migration_add_status_for_tickets.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import * as Knex from 'knex'
+
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table('tickets', table => {
+    table.enu('status',
```
Comment (OmTheTurtle) [inline migrations/20200828113656_migration_add_status_for_tickets.ts:6]: I guess this should be `table.enum`

File: src/components/tickets/ticket.ts [hunk 2]
```diff
@@ -1,9 +1,17 @@
 import { Model } from 'objection'
 
+export enum statusType {
```
Comment (OmTheTurtle) [inline src/components/tickets/ticket.ts:3]: In JS it's a convention to use PascalCase for enums

File: migrations/20200828113656_migration_add_status_for_tickets.ts (hunk 1 above)
Comment (nmate980829) [reply migrations/20200828113656_migration_add_status_for_tickets.ts:6]: "Adds a enum column, (aliased to enu, as enum is a reserved word in JavaScript)."

File: src/components/tickets/ticket.ts (hunk 2 above)
Comment (nmate980829) [reply src/components/tickets/ticket.ts]: Thanks for the advice. I changed it.

**Review by OmTheTurtle — APPROVED** [review]

**Review by triszt4n — COMMENTED** [review]
Nice code! Question: Would it be okay, if we the tickets archived would be at the and at the list given back by the server? And we could even omit sending the archived tickets for the non-admin users. Opinions?
Also, bulma is a bitch, but could you take a look at the ticket's header in mobile view? It collapses.

**Comment by OmTheTurtle** [comment]
I think a filtering option (allowing multiple options at once, like only show "in progress" and "done") would be the best solution with defaulting to "in progress" and "sent".

**Comment by nmate980829** [comment]
> Nice code! Question: Would it be okay, if we the tickets archived would be at the and at the list given back by the server? And we could even omit sending the archived tickets for the non-admin users. Opinions?

> I think a filtering option (allowing multiple options at once, like only show "in progress" and "done") would be the best solution with defaulting to "in progress" and "sent".

It is a nice idea. I think it would make a perfect first issue

**Comment by nmate980829** [comment]
> Also, bulma is a bitch, but could you take a look at the ticket's header in mobile view? It collapses.

I will see into it. Thanks for the help.

**Review by OmTheTurtle — APPROVED** [review]

**Pull request merged**